library/src/main/java/com/woxthebox/draglistview/BoardView.java

```diff
@@ -66,6 +68,8 @@
     private int mDragStartRow;
     private boolean mHasLaidOut;
     private boolean mDragEnabled = true;
+    private Integer mLastDragColumn;
+    private Integer mLastDragRow;
```

Comment:
I rather use primitive int instead of Integer. Init them to RecyclerView.NO_POSITION.